catppuccin_tmux.conf

```diff
@@ -208,7 +208,7 @@ set -ogqF @catppuccin_window_current_right_separator "#{@catppuccin_window_right
       "#{E:@_ctp_w_text_style}#{E:@catppuccin_window_current_left_separator}#{E:@_ctp_w_text_style}#{@catppuccin_window_current_text}#{@_ctp_w_flags}"
     set -agF window-status-current-format "#{E:@catppuccin_window_current_middle_separator}"
     set -agF window-status-current-format \
-      "#{E:@_ctp_w_number_style} #{@catppuccin_window_current_number}#{E:@catppuccin_window_current_right_separator}"
+      "#{E:@_ctp_w_number_style}#{@catppuccin_window_current_number}#{E:@catppuccin_window_current_right_separator}"
```

Comment:
Did you change the default for this as well?

Reply:
Sorry about that! Updated!